examples/chants/components/chants/chant.css.js

```diff
@@ -0,0 +1,39 @@
+import css from "https://tfl.dev/@truffle/utils@~0.0.3/css/css.ts";
+
+export default css`
```

Comment:
should be able to do chant.scss.js and scss instead of css here. and sass compiler shuld get these other prefixed versions you have